aiida_testing/mock_code/_fixtures.py

```diff
@@ -8,33 +8,91 @@
 import inspect
 import pathlib
 import typing as ty
-
+import click
 import pytest
 
+from aiida.orm import Code
+
 from ._env_keys import EnvKeys
-from .._config import get_config
+from .._config import Config, CONFIG_FILE_NAME, ConfigActions
+
+__all__ = (
+    "pytest_addoption",
+    "testing_config_action",
+    "mock_regenerate_test_data",
+    "testing_config",
+    "mock_code_factory",
+)
+
+
+def pytest_addoption(parser):
+    """Add pytest command line options."""
+    parser.addoption(
+        "--testing-config-action",
+        type=click.Choice((c.value for c in ConfigActions)),
```

Comment:
Neat, I didn't know you can use `click.Choice` in combination with `pytest` options 😃